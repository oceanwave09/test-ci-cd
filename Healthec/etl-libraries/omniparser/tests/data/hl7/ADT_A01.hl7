MSH|^~\&|EPIC_CSN|HEALTHEC|||20220625060000|120666|ADT^A01|955530|D|2.3
EVN|A01|20220920010000||APPT_CHECK_IN|120666^First02^Last02^A^^^^^BH^^^^^SCSI||
PID|1|36914001|PPA123654001|MEM1009201|First01^Last01^M||19710101|F||1004-1|343 Thornall Street^^Edison^NJ^08837^US^^^MACOMB|08837|(732)271-0600^PRN^PH^^^^^^||ENGLISH|D||92000072018||||2148-5||||||||N|
PV1|1|I|SCSI^^^HEALTHEC^^^^^^^DEPID||||1962411111^First02^Last02^^^^MD|||MED|||||||960^Last03^^^^MD||83732788|||||||||||||||||||||||||20220920010000|
OBX|1|NM|1100061^HGB^L^718-7^Hgb Bld-mCnc^LN|1|14.2|g/dL|14.0-18.0|L|||F|||20220922231234||
OBX|2|NM|1100059^HCT^L^4544-3^Hct Fr Bld Auto^LN|1|42.2|%|42.0-52.0|L|||F|||20220922231234||
OBX|3|NM|1100068^MCV^L^787-2^MCV RBC Qn Auto^LN|1|94.1|fL|78.0-98.0|L|||F|||20220922231234||
OBX|4|NM|1100066^MCH^L^785-6^MCH RBC Qn Auto^LN|1|31.2|pg|27.0-31.0|H|||F|||20220922231234||
DG1|1|ICD-10-CM|E78.5^Hyperlipidemia, unspecified^ICD-10-CM|Hyperlipidemia, unspecified|20220922231234|W||||||||||||||||||||
DG1|2|ICD-10-CM|W59.11XA^Bitten by nonvenomous snake, initial encounter^ICD-10-CM|Bitten by nonvenomous snake, initial encounter||W||||||||||||||||||||
IN1|1|1022^BCBSM PPO|1004|BCBSM|343 Thornall Street^^Edison^NJ^08837^|||Y123654||||20190307|20221222|||First01^Last01^M^|01|19710101|343 Thornall Street^^Edison^NJ^08837^US^^^MACOMB|||1|||YES||||20220920010000|XTADTLABREG^TEST^ADT^LAB USER^|||||171|PPA123654001||||||1|F|343 Thornall Street^^Edison^NJ^08837^US|MESSAGE SENT||BOTH|
IN2||111-11-1111|||Payor||||||||||||||||||||||||||||||||||||||||||||||||||||||||PPA123654001||(732)271-0600^PRN^PH^^^^^^|(732)271-0600^PRN^PH^^^^^^||||||CHRYSLER GROUP
