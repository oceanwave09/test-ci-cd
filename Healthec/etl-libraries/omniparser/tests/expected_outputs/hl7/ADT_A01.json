[
	{
		"DG1": [
			{
				"diagnosis_code_identifier": "E78.5",
				"diagnosis_code_system": "ICD-10-CM",
				"diagnosis_code_text": "Hyperlipidemia, unspecified",
				"diagnosis_coding_method": "ICD-10-CM",
				"diagnosis_date_time": "20220922231234",
				"diagnosis_description": "Hyperlipidemia, unspecified",
				"diagnosis_type": "W",
				"set_id": "1"
			},
			{
				"diagnosis_code_identifier": "W59.11XA",
				"diagnosis_code_system": "ICD-10-CM",
				"diagnosis_code_text": "Bitten by nonvenomous snake, initial encounter",
				"diagnosis_coding_method": "ICD-10-CM",
				"diagnosis_description": "Bitten by nonvenomous snake, initial encounter",
				"diagnosis_type": "W",
				"set_id": "2"
			}
		],
		"EVN": {
			"event_reason_code": "APPT_CHECK_IN",
			"event_type_code": "A01",
			"operator_id": "120666^First02^Last02^A^^^^^BH^^^^^SCSI",
			"recorded_date_time": "20220920010000"
		},
		"MSH": {
			"date_time_of_message": "120666",
			"encoding_characters": "EPIC_CSN",
			"field_seperator": "^~\\&",
			"message_control_id": "D",
			"message_type": "955530",
			"processing_id": "2.3",
			"receiving_facility": "20220625060000",
			"security": "ADT^A01",
			"sending_application": "HEALTHEC"
		},
		"OBX": [
			{
				"abnormal_flags": "L",
				"date_time_of_the_observation": "20220922231234",
				"observation_alternate_coding_system": "LN",
				"observation_alternate_identifier": "718-7",
				"observation_alternate_text": "Hgb Bld-mCnc",
				"observation_identifier": "1100061",
				"observation_identifier_system": "L",
				"observation_identifier_text": "HGB",
				"observation_result_status": "F",
				"observation_sub_identifier": "1",
				"observation_value": "14.2",
				"reference_range": "14.0-18.0",
				"set_id_obx": "1",
				"units": "g/dL",
				"value_type": "NM"
			},
			{
				"abnormal_flags": "L",
				"date_time_of_the_observation": "20220922231234",
				"observation_alternate_coding_system": "LN",
				"observation_alternate_identifier": "4544-3",
				"observation_alternate_text": "Hct Fr Bld Auto",
				"observation_identifier": "1100059",
				"observation_identifier_system": "L",
				"observation_identifier_text": "HCT",
				"observation_result_status": "F",
				"observation_sub_identifier": "1",
				"observation_value": "42.2",
				"reference_range": "42.0-52.0",
				"set_id_obx": "2",
				"units": "%",
				"value_type": "NM"
			},
			{
				"abnormal_flags": "L",
				"date_time_of_the_observation": "20220922231234",
				"observation_alternate_coding_system": "LN",
				"observation_alternate_identifier": "787-2",
				"observation_alternate_text": "MCV RBC Qn Auto",
				"observation_identifier": "1100068",
				"observation_identifier_system": "L",
				"observation_identifier_text": "MCV",
				"observation_result_status": "F",
				"observation_sub_identifier": "1",
				"observation_value": "94.1",
				"reference_range": "78.0-98.0",
				"set_id_obx": "3",
				"units": "fL",
				"value_type": "NM"
			},
			{
				"abnormal_flags": "H",
				"date_time_of_the_observation": "20220922231234",
				"observation_alternate_coding_system": "LN",
				"observation_alternate_identifier": "785-6",
				"observation_alternate_text": "MCH RBC Qn Auto",
				"observation_identifier": "1100066",
				"observation_identifier_system": "L",
				"observation_identifier_text": "MCH",
				"observation_result_status": "F",
				"observation_sub_identifier": "1",
				"observation_value": "31.2",
				"reference_range": "27.0-31.0",
				"set_id_obx": "4",
				"units": "pg",
				"value_type": "NM"
			}
		],
		"PID": {
			"alternate_patient_id": "MEM1009201",
			"county_code": "08837",
			"date_time_of_birth": "19710101",
			"marital_status_identifier": "D",
			"patient_account_number": "92000072018",
			"patient_address_county_code": "MACOMB",
			"patient_city": "Edison",
			"patient_country": "US",
			"patient_death_indicator": "N",
			"patient_ethnic_group_identifier": "2148-5",
			"patient_family_name": "First01",
			"patient_given_name": "Last01",
			"patient_identifier_id": "36914001",
			"patient_identifier_list_id": "PPA123654001",
			"patient_middle_name": "M",
			"patient_postal_code": "08837",
			"patient_race_identifier": "1004-1",
			"patient_state": "NJ",
			"patient_street_address": "343 Thornall Street",
			"phone_number_home": "(732)271-0600",
			"phone_number_home_code": "PRN",
			"phone_number_home_equipment_type": "PH",
			"primary_language_identifier": "ENGLISH",
			"set_id_pid": "1",
			"sex": "F"
		},
		"PV1": {
			"admit_date_time": "20220920010000",
			"admitting_doctor_family_name": "Last03",
			"admitting_doctor_id": "960",
			"admitting_doctor_preffix": "MD",
			"assigned_patient_facility": "HEALTHEC",
			"assigned_patient_location_authority": "DEPID",
			"assigned_patient_point_of_care": "SCSI",
			"attending_doctor_degree": "MD",
			"attending_doctor_family_name": "First02",
			"attending_doctor_given_name": "Last02",
			"attending_doctor_id": "1962411111",
			"hospital_service": "MED",
			"patient_class": "I",
			"set_id": "1",
			"visit_id_number": "83732788"
		},
		"insurance": [
			{
				"IN1": {
					"company_plan_code": "171",
					"coord_of_ben_priority": "1",
					"coverage_type": "BOTH",
					"group_number": "Y123654",
					"health_plan_identifier": "1022",
					"health_plan_text": "BCBSM PPO",
					"insurance_company_city": "Edison",
					"insurance_company_identifier": "1004",
					"insurance_company_name": "BCBSM",
					"insurance_company_state": "NJ",
					"insurance_company_street_address": "343 Thornall Street",
					"insurance_company_zip": "08837",
					"insured_city": "Edison",
					"insured_country": "US",
					"insured_county": "MACOMB",
					"insured_family_name": "First01",
					"insured_given_name": "Last01",
					"insured_initial": "M",
					"insured_state": "NJ",
					"insured_street_address": "343 Thornall Street",
					"insured_zip": "08837",
					"insureds_administrative_sex": "F",
					"insureds_date_of_birth": "19710101",
					"insureds_employers_address": "343 Thornall Street^^Edison^NJ^08837^US",
					"insureds_employment_status": "1",
					"insureds_relationship_identifier": "01",
					"plan_effective_date": "20190307",
					"plan_expiration_date": "20221222",
					"policy_number": "PPA123654001",
					"report_of_eligibility_flag": "YES",
					"set_id_in1": "1",
					"verification_by": "XTADTLABREG^TEST^ADT^LAB USER^",
					"verification_date_time": "20220920010000",
					"verification_status": "MESSAGE SENT"
				},
				"IN2": {
					"insured_employer_organization_name_and_id": "CHRYSLER GROUP",
					"insureds_employer_phone_number": "(732)271-0600^PRN^PH^^^^^^",
					"insureds_phone_number_home": "(732)271-0600^PRN^PH^^^^^^",
					"insureds_social_security_number": "111-11-1111",
					"mail_claim_party": "Payor",
					"patient_member_number": "PPA123654001"
				}
			}
		],
		"row_id": "03e585e2-ec9d-4365-b44a-3359d44c3db8"
	}
]
